walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/random.html

```diff
@@ -3,8 +3,9 @@
   <head>
     <meta charset="UTF-8">
     <title>Random Stanley 🐈</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <link rel="stylesheet" href="style.css"> 
+    <!-- Seems like the import for randomizeImage isn't working-->
```

Comment:
This comment is confusing to me without context since I don't see randomizeImage referenced anywhere on this page. Can you add more context?

Reply:
Sorry I left this comment in for myself while troubleshooting an import problem but I forgot to take it out!